### PR TITLE
Update to OVTF1.1.0, GATK4.2.4.1, TF2.7.0

### PR DESCRIPTION
**Pull request by ravi9**
Hi @Qi77Qi, @rtitle 
OVTF1.1.0 is released, we updated the dockerfile.
- Along with the OVTF update, we updated with GATK-4.2.4.1, TF-2.7.0.
- GPU support is enabled. We added notes on GPU support in the README.
- The end-user options are either:  `OVTF with CPUs` or `Native TF with CPUs/GPUs`

CC: @cavusmustafa

### CODE REVIEW

**Comment by ravi9** [comment]
Hi @Qi77Qi, We have tested this image and it works fine. Could review and merge it.
Also, @rtitle,  GPU support works. See the README section: Note on CUDA-enabled GPU devices Support.

File: terra-jupyter-gatk-ovtf/Dockerfile [hunk 1]
```diff
@@ -29,28 +29,40 @@ RUN apt-get update && apt-get install -yq --no-install-recommends \
...
+ENV KERAS_VER 2.7.0
+ENV OVTF_VER 1.1.0
+ENV TF_ENABLE_ONEDNN_OPTS 1
+
 ENV PIP_USER=false
 
-# Force reinstalling TF-2.5.1, Keras-2.5.0 to satisfy OVTF compatibility.
-RUN pip install --force-reinstall tensorflow==2.5.1 keras==2.5.0rc0
+# Force reinstalling TF and Keras to satisfy OVTF compatibility.
+RUN pip install --force-reinstall tensorflow==$TF_VER keras==$KERAS_VER
```
Comment (Qi77Qi) [inline terra-jupyter-gatk-ovtf/Dockerfile:48]: did you check if forcing a new install of TF breaks GPU support?

Comment (ravi9) [reply terra-jupyter-gatk-ovtf/Dockerfile:48]: We added a note on GPU support in the [README](https://github.com/ravi9/terra-docker/tree/ovtf-update-1.1/terra-jupyter-gatk-ovtf#note-on-cuda-enabled-gpu-devices-support)

Please verify like below.
import tensorflow as tf
# CPU and GPU support available. openvino_tensorflow is disabled
tf.config.list_physical_devices()

**Comment by xuscrbw** [comment]
Hi @ravi9, 
Would you be able to add the changes to the: config/conf.json and terra-jupyter-gatk-ovtf/CHANGELOG.md from this PR: https://github.com/DataBiosphere/terra-docker/pull/293

into this one?

The tests have passed on our side, so I think we should be good to merge. We want to keep the commit history accurate (as opposed to copy pasting into a new branch).

Thanks, let me know if you need any clarification!

Sincerely,
Wilson

**Review by xuscrbw — APPROVED** [review]
LGTM

**Pull request merged**